### PR TITLE
Updates for demo

### PR DESCRIPTION
**Pull request by GeorgeWalker**

### CODE REVIEW

File: spec/example-artifacts.json [hunk 1]
```diff
@@ -38,12 +49,12 @@
...
       "provides": [
+	    "server" :
         {
-          "interface": "ca.bc.gov.nrs.titan.nrs-titan-ds",
-          "layer":"server"
+          "interface": "ca.bc.gov.nrs.titan.nrs-titan-ds"          
         }
       ],
-      "requires": {
+      "requires": [
```
Comment (GeorgeWalker) [inline spec/example-artifacts.json]: Should be "requires" : {

File: OpenShift/Templates/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+AGRI-NMP
```
Comment (cvarjao) [inline OpenShift/Templates/README.md]: remove AGRI reference

File: cmdb.iml [hunk 3]
```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_6">
+  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_7">
```
Comment (cvarjao) [inline cmdb.iml]: you may increase to JDK8

**Review by cvarjao — COMMENTED** [review]
Use a reference to JQuery from a public CDN

**Pull request merged**